### PR TITLE
Rescan for dumps

### PR DESCRIPTION
**Pull request by mkestler-mozilla**
This is a working theory that the dump files might not be written at the time the crash reporter is launched.  There is a 5x retry of the directory listing with an increasing interval (+ 1s each time).

### CODE REVIEW

File: src/crashreporter/crashreporterapp.cpp [hunk 1]
```diff
@@ -11,10 +11,6 @@
 #include "crashreporterfactory.h"
 
 int CrashReporterApp::main(int argc, char* argv[]) {
-#if QT_VERSION < 0x060000
-  // This flag is set by default in qt6.
-  QApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
-#endif
```
Comment (bakulf) [inline src/crashreporter/crashreporterapp.cpp:17]: Unrelated?

Comment (mkestler-mozilla) [reply src/crashreporter/crashreporterapp.cpp:17]: Yes, but since we are all in on Qt6, I thought I'd pull it.
